stylesheets/utils/utilities.css

```diff
@@ -135,3 +143,7 @@ h6 {
   display: flex;
   flex-direction: column;
 }
+
+.gap-sm{
+    gap: .5rem;
```

Comment:
Mostly class names changed and minor changes. So no suggestions. Everything looks good to me.